### PR TITLE
Output files metadata

### PR DESCRIPTION
**Pull request by markx**

### CODE REVIEW

File: provider/bitmovin/bitmovin.go [hunk 1]
```diff
@@ -987,6 +992,56 @@ func (p *bitmovinProvider) addOutputStatusInfo(job *db.Job, status *provider.Job
...
+	var files []provider.OutputFile
+	for _, muxing := range muxings {
+		encodingS := services.NewEncodingService(p.client)
+		resp, err := encodingS.RetrieveMP4MuxingInformation(job.ProviderJobID, stringValue(muxing.ID))
+		if err != nil {
+			return err
+		}
+
+		info := resp.Data.Result
+		if len(info.VideoTracks) > 0 {
```
Comment (markx) [inline provider/bitmovin/bitmovin.go]: Should it return error if `len(VideoTracks) == 0`?

Comment (fsouza) [reply provider/bitmovin/bitmovin.go]: Yeah, since this is a required info, let's fail in case it's missing.

File: provider/bitmovin/bitmovin.go [hunk 2]
```diff
@@ -987,28 +992,83 @@ func (p *bitmovinProvider) addOutputStatusInfo(job *db.Job, status *provider.Job
...
 	if err != nil {
 		return err
 	}
-	if len(resp.Data.Result.Items) > 0 {
-		streamID := stringValue(resp.Data.Result.Items[0].ID)
-		streamInput, err := encodingS.RetrieveStreamInputData(job.ProviderJobID, streamID)
+
+	var files []provider.OutputFile
+	for _, muxing := range muxings {
+		encodingS := services.NewEncodingService(p.client)
```
Comment (fsouza) [inline provider/bitmovin/bitmovin.go]: Should we create ``encodingS`` outside the forloop?

Comment (markx) [reply provider/bitmovin/bitmovin.go]: Good catch!   It's a copy-paste error.

**Pull request merged**